Example/Source/Examples/Scrolling/CardScrollingViewController.swift

```diff
@@ -20,14 +20,14 @@ class CardScrollingViewController: BaseScrollingViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        layout.zIndexBehavior = .BottomUp
+        layout.zIndexBehavior = .bottomUp
         behavior = CardLayoutBehavior(dataSource: self)
     }
 
 }
 
 extension CardScrollingViewController: CardLayoutBehaviorDataSource {
-    func cardLayoutBehavior(behavior: CardLayoutBehavior, smallHeightForItemAtIndexPath indexPath: NSIndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGFloat? {
+    func cardLayoutBehavior(_ behavior: CardLayoutBehavior, smallHeightForItemAtIndexPath indexPath: IndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGFloat? {
```

Comment:
This should be
`func cardLayoutBehavior(_ behavior: CardLayoutBehavior, smallHeightForItemAt indexPath: IndexPath, with identifier: String, in collectionViewLayout: UICollectionViewLayout) -> CGFloat? {`